src/houghtransform.jl

```diff
@@ -37,13 +43,18 @@ julia> lines = hough_transform_standard(img_edges, 1, linspace(0,π,30), 40, 5)
 ```
 """  
 function hough_transform_standard(
-    img::AbstractArray{T,2},
-    ρ::Real, θ::AbstractRange,
-    threshold::Integer, linesMax::Integer) where T<:Union{Bool,Gray{Bool}}
+    img::AbstractMatrix;
+    stepsize=1,
+    angles=range(0,stop=pi,length=minimum(size(img))),
+    vote_threshold=minimum(size(img)) / stepsize -1,
+    max_linecount=typemax(Int))
```

Comment:
If you're removing the `eltype` restriction, we need tests on other eltypes (e.g., `Gray{N0f8}`, `RGB{Float32}`).

Reply:
Ah good catch, no `img` should be the edges, e.g. `Bool`.